src/assets/code_of_conduct.md

```diff
@@ -0,0 +1,92 @@
+- Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language. 
+- Posting or displaying sexually explicit or violent material. 
+- Posting or threatening to post other people’s personally identifying information ("doxing"). 
+- Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability. 
+- Inappropriate photography or recording. 
+- Inappropriate physical contact. You should have someone’s consent before touching them. 
+- Unwelcome sexual attention. This includes sexualized comments or jokes; inappropriate touching, groping, and unwelcome sexual advances. 
+- Deliberate intimidation, stalking or following (online or in person). 
+- Advocating for, or encouraging, any of the above behavior. 
+- Sustained disruption of community events, including talks and, presentations. 
+
+Consequences of Unacceptable Behavior
+-------------------------------------
+Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated. 
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately. If a community member engages in unacceptable behavior, Windy City DevFest organizers may take any action they deem appropriate, up to and including a warning, temporary ban, or permanent expulsion from the community/event without warning (and without refund in the case of a paid event).
+
+Reporting
+---------
+If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. Conference staff can be identified by special lanyards and/or special badges. Conference staff is also located at the registration desk throughout the duration of the event. Harassment and other code of conduct violations reduce the value of our event for everyone. We want you to be happy at our event. People like you make our event a better place.
+
+You can make a report either personally or anonymously.
+
+Anonymous Report
+----------------
+You can make an anonymous report here:
+
+[https://bit.ly/2RstTJ8](https://bit.ly/2RstTJ8)
+
+We can't follow up an anonymous report with you directly, but we will fully investigate it and take whatever action is necessary to prevent a recurrence.
+
+Personal Report
+---------------
+You can make a personal report by getting in contact with any of the event organizers. Event Organizers can be identified by special lanyards and/or special badges. If you wish, you can also use the anonymous form but provide your contact information at the end. In the event organizers are not able to be reached, please reach out to us directly by phone/email at:
+
+- General GDG/DevFest Organizer: – ([chicagogdg@gmail.com](mailto:chicagogdg@gmail.com)) 
+- Co-Organizer: Joel Vasallo – 773 318 8144 ([joelvasallo@gmail.com](mailto:joelvasallo@gmail.com)) 
+- WTM Lead/ Co-Organizer: <> – <> ([placeholder@exmaple.com](mailto:placeholder@example.com))
```

Comment:
Probably want to put in Deana in here for us, worry about templating later.

Reply:
Done!